apps/settings/js/vertical_homescreen.js

```diff
@@ -1,49 +0,0 @@
-'use strict';
```

Comment:
As soon as we are removing this module completely, we shall also delete unit test covering it, otherwise we will get test failed result on while landing to Mozilla.

The affected test is located in ./test/unit/vertical_homescreen_test.js